src/OpenTelemetry.Instrumentation.Runtime/README.md

```diff
@@ -316,15 +296,6 @@ Relevant API:
   Occurs when an exception is thrown in managed code, before the runtime searches
   the call stack for an exception handler in the application domain.
 
-## Troubleshooting
```

Comment:
I'd suggest keeping the section.
We can add quick notes about.
if a metric is missing, review the doc to see if the metric is available in .NET version you are running.
also mention those metrics which are only available after GC has be completed atleast once?